README.md

```diff
@@ -100,7 +100,7 @@ If you look at your `src/test.ts` (or similar bootstrapping test file) file you'
 
 ## Snapshot testing
 
-**Since version 1.1.0** it's possible to [snapshot test](http://facebook.github.io/jest/docs/snapshot-testing.html#snapshot-testing-with-jest) your Angular components. Please note it's still under active development and may be a subject of change. You can lookup [example](/example/src/app) for details
+**Since version 1.1.0** it's possible to [snapshot test](http://facebook.github.io/jest/docs/snapshot-testing.html#snapshot-testing-with-jest) your Angular components. Please note it's still under active development and may be a subject of change. You can lookup [example](/e2e/example/src/app) for details
```

Comment:
Again, I would like to move away from the `example` vocabulary

Reply:
done